open_academy/views/session_views.xml

```diff
@@ -10,4 +10,42 @@
 
     <menuitem id="session_menu" name="Sessions" sequence="1"
     action="session_action" parent="open_academy_root_session_menu"/>
```

Comment:
![image](https://user-images.githubusercontent.com/47753811/181607970-86ad142a-fa71-43c7-a2a2-ebb5e2d57a8c.png)

Integrate the menus in a single one, you should not have an application for each model, for that there are submenus.

Reply:
Done